modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,55 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory pages in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which updates the {op-system} kernel. You can create a custom machine config pool to add 64k pages to your nodes. 
```

Comment:
"This is done by the Machine Config Operator (MCO) using machine config pools which updates the {op-system} kernel" ?

Reply:
Maybe I can word that better. I thought the MCO is the operator that updates the kernel. Like how we would make a machinepool and the MCO will use the machinepool to update the kernel. Or is that not accurate?

Reply:
correct. so the MCO uses the machineconfigpool to update  the kernel.